include/ensmallen_bits/cmaes/cmaes_impl.hpp

```diff
@@ -216,8 +216,7 @@ typename MatType::elem_type CMAES<SelectionPolicyType>::Optimize(
     }
 
     const ElemType psNorm = arma::norm(ps[idx1]);
-    sigma(idx1) = sigma(idx0) * std::pow(
-        std::exp(cs / ds * psNorm / enn - 1), 0.3);
+    sigma(idx1) = sigma(idx0) * std::exp(cs / ds * psNorm / enn - 1);
```

Comment:
Should this not be 
```suggestion
    sigma(idx1) = sigma(idx0) * std::exp(cs / ds *( psNorm / enn - 1));
```

Reply:
Oh! thanks for pointing that out @favre49